chemprop/nn/__init__.py

```diff
@@ -1,73 +1,73 @@
 from .agg import (
     Aggregation,
     AggregationRegistry,
+    AttentiveAggregation,
     MeanAggregation,
-    SumAggregation,
     NormAggregation,
-    AttentiveAggregation,
+    SumAggregation,
 )
 from .loss import (
-    LossFunction,
-    LossFunctionRegistry,
-    MSELoss,
-    BoundedMSELoss,
-    MVELoss,
-    EvidentialLoss,
     BCELoss,
-    CrossEntropyLoss,
-    MccMixin,
+    BinaryDirichletLoss,
     BinaryMCCLoss,
-    MulticlassMCCLoss,
+    BoundedMSELoss,
+    CrossEntropyLoss,
     DirichletMixin,
-    BinaryDirichletLoss,
+    EvidentialLoss,
+    LossFunction,
+    LossFunctionRegistry,
+    MccMixin,
+    MSELoss,
     MulticlassDirichletLoss,
+    MulticlassMCCLoss,
+    MVELoss,
     SIDLoss,
     WassersteinLoss,
 )
+from .message_passing import (
+    AtomMessagePassing,
+    BondMessagePassing,
+    MessagePassing,
+    MulticomponentMessagePassing,
+)
 from .metrics import (
-    Metric,
-    MetricRegistry,
-    ThresholdedMixin,
-    MAEMetric,
-    MSEMetric,
-    RMSEMetric,
-    BoundedMixin,
+    BCEMetric,
+    BinaryAccuracyMetric,
+    BinaryAUPRCMetric,
+    BinaryAUROCMetric,
+    BinaryF1Metric,
+    BinaryMCCMetric,
     BoundedMAEMetric,
+    BoundedMixin,
     BoundedMSEMetric,
     BoundedRMSEMetric,
-    R2Metric,
-    BinaryAUROCMetric,
-    BinaryAUPRCMetric,
-    BinaryAccuracyMetric,
-    BinaryF1Metric,
-    BCEMetric,
     CrossEntropyMetric,
-    BinaryMCCMetric,
+    MAEMetric,
+    Metric,
+    MetricRegistry,
+    MSEMetric,
     MulticlassMCCMetric,
+    R2Metric,
+    RMSEMetric,
     SIDMetric,
+    ThresholdedMixin,
     WassersteinMetric,
 )
-from .message_passing import (
-    MessagePassing,
-    AtomMessagePassing,
-    BondMessagePassing,
-    MulticomponentMessagePassing,
-)
 from .predictors import (
-    Predictor,
-    PredictorRegistry,
-    RegressionFFN,
-    MveFFN,
-    EvidentialFFN,
-    BinaryClassificationFFNBase,
     BinaryClassificationFFN,
+    BinaryClassificationFFNBase,
     BinaryDirichletFFN,
+    EvidentialFFN,
     MulticlassClassificationFFN,
     MulticlassDirichletFFN,
+    MveFFN,
+    Predictor,
+    PredictorRegistry,
+    RegressionFFN,
     SpectralFFN,
 )
-from .utils import Activation
 from .transforms import UnscaleTransform
+from .utils import Activation
 
 __all__ = [
     "Aggregation",
```

Comment:
Originally the order of __all__ matched the order of imports? These changes breaks that symmetry, are we okay with that or should the order in __all__ also be updated?

Reply:
This is better IMO. The imports are in sorted order, but these are _separate_ from the definitions this module _exports_. We should want the exports to also be in sorted order.

Reply:
This will be fixed when https://github.com/PyCQA/isort/pull/2283 is merged and released